package.json

```diff
@@ -337,6 +338,11 @@
                 "category": "Cosmos DB",
                 "command": "cosmosDB.openStoredProcedure",
                 "title": "Open Stored Procedure"
+            },
+            {
+                "category": "PostgreSQL",
+                "command": "cosmosDB.getPostgresCredentials",
+                "title": "Enter Server Credentials"
```

Comment:
The old title was redundant: 
<img width="509" alt="Screen Shot 2020-03-26 at 10 42 59 AM" src="https://user-images.githubusercontent.com/22795803/77678788-e8145400-6f4e-11ea-8bbe-7a8001f00145.png">

New title:
<img width="509" alt="Screen Shot 2020-03-26 at 10 42 27 AM" src="https://user-images.githubusercontent.com/22795803/77678700-c7e49500-6f4e-11ea-8021-5ea82aba86f9.png">